server/config/defaults.js

```diff
@@ -27,6 +27,7 @@ function getDefaults () {
     // core modules
     account: {},
     admin: {},
-    store: {}
+    store: {},
+    plugins: []
```

Comment:
I'm sorry I could figure this out by myself I’m just busy conferencing right now, I'll have more time again on Friday - Monday. 

Can you explain why we set `plugins: []` instead of `plugins: {}`? Isn’t the final internal format an object for `config.plugins`? Then we wouldn’t need this in `parse-options.js` I think:

```diff
 +  // ensure we have all required options
+  _.defaultsDeep(options, {
+    plugins: {}
+  })
```

And instead of `var plugins = Object.keys(pkg.hoodie.plugins).map(function (key) {` we would set it to an object, with the plugin names being the keys?

Am I missing something?